pkgs/stdenv/linux/bootstrap-files/powerpc64le.nix

```diff
@@ -0,0 +1,25 @@
+#
+# Files came from this Hydra build:
+#
+#   https://hydra.nixos.org/build/172142499
+#
+# Which used nixpkgs revision 49a83445c28c4ffb8a1a90a1f68e6150ea48893b
+# to instantiate:
+#
+#   /nix/store/gj272sd56gsj6qpyzh4njpfzwdhviliz-stdenv-bootstrap-tools-powerpc64le-unknown-linux-gnu.drv
+#
+# and then built:
+#
+#   /nix/store/n81pljbd8m0xgypm84krc2bnvqgjrfxx-stdenv-bootstrap-tools-powerpc64le-unknown-linux-gnu
+#
+{
+  busybox = import <nix/fetchurl.nix> {
+    url = "http://tarballs.nixos.org/stdenv-linux/powerpc64le/49a83445c28c4ffb8a1a90a1f68e6150ea48893b/busybox";
+    sha256 = "sha256-UscnfGKOZAKLkPcRtwrbT5Uj8m3Kj9jhkKp9MUc1eCY=";
```

Comment:
Shouldn't this have `sha256` hash `6bc05832180f5075f4458c32eb0a5e2b673f605884dce01822be474f4e0a63ee` which would be `sha256-a8BYMhgPUHX0RYwy6wpeK2c/YFiE3OAYIr5HT04KY+4=` in SRI representation?

Reply:
The executable bit affects nix's hash, but it doesn't affect `sha256sum`'s hash.

Took me a long time to figure that one out when I did the mips64 bootstrap :)

Reply:
Indeed, `nix-hash` gives the correct hash for me as well!